client/src/models/enums/DialogStatus.ts

```diff
@@ -0,0 +1,7 @@
+enum DialogOptions {
```

Comment:
Where do you use this enum?

Reply:
This file was deleted, will be fixed on the new pr